### DISCUSSION TITLE
2024-08-27 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#367] [Meetings]
Please review our meeting agenda here: 
[UCUM Committee_Meeting Agenda_2024-08-27_FINAL](https://docs.google.com/document/d/1nsBdD_3Je8b_i5CqsJYABZ5xZTFSebZ1/edit)

Reference materials:
[UCUM New Unit Template_2024-01.xlsx](https://docs.google.com/spreadsheets/d/1xBGaNLuMaJBFnFdi81ezDF5oHzN_Ozww/edit?usp=drive_link&ouid=110875911742539584915&rtpof=true&sd=true)

[UCUM Committee_Meeting Summary_2024-06-25_FINAL.docx](https://docs.google.com/document/d/1xzBeDJvi1Zl9fiFQQwg2qM3ldgC4ZQ2A/edit) 

All active GitHub issues/proposals https://github.com/ucum-org/ucum/issues

@ucum-org/regenstrief
@ucum-org/advisors

**Comment by ehafeza** [comment]
Attached are the references for IGE discussion.

[IgE allergy diagnostics.pdf](https://github.com/user-attachments/files/16764220/IgE.allergy.diagnostics.pdf)
[Molecular allergy diagnostics using IgE singleplex determinations_methodological and practical considerations for use in clinical routine.pdf](https://github.com/user-attachments/files/16764223/Molecular.allergy.diagnostics.using.IgE.singleplex.determinations_methodological.and.practical.considerations.for.use.in.clinical.routine.pdf)
[Precision medicine allergy immunoassay methods for assessing immunoglobulin E sensitization to aeroallergen molecules.pdf](https://github.com/user-attachments/files/16764224/Precision.medicine.allergy.immunoassay.methods.for.assessing.immunoglobulin.E.sensitization.to.aeroallergen.molecules.pdf)
[Quantitation of Mouse-Human Chimeric Allergen Specific IgEAntibodies with ImmunoCAP Technology.pdf](https://github.com/user-attachments/files/16764225/Quantitation.of.Mouse-Human.Chimeric.Allergen.Specific.IgEAntibodies.with.ImmunoCAP.Technology.pdf)
[Quantitative IgE antibody assays in allergic disease.pdf](https://github.com/user-attachments/files/16764226/Quantitative.IgE.antibody.assays.in.allergic.disease.pdf)
[Quantitative Measurement of Allergen-Specific Immunoglobulin E Levels in Mass Units.pdf](https://github.com/user-attachments/files/16764227/Quantitative.Measurement.of.Allergen-Specific.Immunoglobulin.E.Levels.in.Mass.Units.pdf)

**Comment by ucumworker** [comment]
-- Please reply above this line --

Hello Ucum-Org/ucum,

We received your email request and will get back to you as soon as we
can. Please know our response time does vary at different times of the
year due to our bi-annual release of LOINC and other events. 

Thank you,

Best Regards,
The Health Data Standards Team
Regenstrief Institute
